src/test/resources/features/11-allergy/allergies.feature

```diff
@@ -5,14 +5,28 @@ Feature: Allergies Management
     And a user clicks on Allergies link
     Then the system loads Allergies page
 
-  @run
   Scenario: Add No known allergies
     And a user clicks No Known Allergy button
     Then the system add no known allergies into the allergies table
     And close browser
 
-  @run
```

Comment:
Same here

Reply:
See the comment above

Reply:
Take a look at this line [no known allergy](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/src/test/resources/features/11-allergy/allergies.feature#L15) and [here ](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/src/test/java/org/openmrs/contrib/qaframework/automation/AllergiesSteps.java#L47) ,so ideally we should keep the @run since these steps had already be written we except cucumber would trigger them

Reply:
Have also Added a ticket to track these up coming changes to  be written ,this should make you keep going  checkout here https://issues.openmrs.org/browse/RATEST-70.

Reply:
> so ideally we should keep the @run since these steps had already be written we except cucumber would trigger them

Your right! Have updated the PR